src/de/jost_net/JVerein/gui/menu/WiedervorlageMenu.java

```diff
@@ -33,6 +36,10 @@ public class WiedervorlageMenu extends ContextMenu
    */
   public WiedervorlageMenu(TablePart table)
   {
+    addItem(new CheckedSingleContextMenuItem("Bearbeiten", new WiedervorlageAction(null),
+        "text-x-generic.png"));
+    addItem(new CheckedSingleContextMenuItem("Mitglied anzeigen",
+        new MitgliedDetailAction(), "user-friends.png"));
```

Comment:
Das Verhalten des Erledigt Button finde ich verwirrend. Einmal setzt er ein Datum und andermal löscht er es.
Da fände ich zwei Menü Einträge besser. Einmal "Erledigung setzen" und "Erledigung löschen". 
Evtl. sollte es auch Erledigungsdatum heißen.
PS: Kursteilnehmer hat auch ein "Abbuchungsdatum löschen" mit Löschen Icon.

Reply:
Stimmt, das dacht ich auch, war aber schon so im code. Da ich auch in #333 hier eine Frage hinzugefügt habe mache ich es da.

Reply:
Jetzt habe ich dort die Frage wieder entfernt. und in diesem PR den neuen Menüeintrag hinzugefügt

Reply:
Warum hast du das Löschen Icon geändert? Da haben wir doch immer den Trash.

Reply:
Das war keine Absicht, habe ich geändert